SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialWorkerFlags.h

```diff
@@ -19,6 +20,7 @@ class SPATIALGDK_API USpatialWorkerFlags : public UBlueprintFunctionLibrary
 	UFUNCTION(BlueprintCallable, Category="SpatialOS")
 	static bool GetWorkerFlag(const FString& Name, FString& OutValue);
 
+	static FOnWorkerFlagsUpdated OnWorkerFlagsUpdated;
```

Comment:
Hmmmm, can you make this blueprint exposed?

Reply:
No. uproperties can't be static

Reply:
@Vatyx added a function to bind to multicast delegate from BP

Reply:
Can you add something to unbind as well?

Reply:
added